<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>63</x>
     <y>83</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>0.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>163</x>
     <y>83</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>0.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>263</x>
     <y>83</y>
     <width>78</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>0.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.7</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>0.9</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1.5</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>153</x>
     <y>243</y>
     <width>98</width>
     <height>27</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Setting</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>163</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>level 2</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_7">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>163</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>level 3</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_4">
   <property name="geometry">
    <rect>
     <x>143</x>
     <y>183</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_5">
   <property name="geometry">
    <rect>
     <x>213</x>
     <y>183</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>10</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>30</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>40</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>70</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>80</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>90</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>100</string>
    </property>
   </item>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>134</x>
     <y>10</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;set weighted value&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>66</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Speed&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Steering&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_4">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>50</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;weather&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_5">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>129</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;set speed standard&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <zorder>pushButton</zorder>
  <zorder>lineEdit_6</zorder>
  <zorder>lineEdit_7</zorder>
  <zorder>textBrowser</zorder>
  <zorder>textBrowser_2</zorder>
  <zorder>textBrowser_3</zorder>
  <zorder>textBrowser_4</zorder>
  <zorder>textBrowser_5</zorder>
  <zorder>comboBox_5</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>comboBox</zorder>
  <zorder>comboBox_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
